### PR TITLE
Resolve host if address not set

### PR DESCRIPTION
**Pull request by micw**
Preparation for https://github.com/Mailu/Mailu/pull/1114

### CODE REVIEW

File: socrate/system.py [hunk 1]
```diff
@@ -18,3 +19,14 @@ def resolve_address(address):
...
+
+def get_host_address_from_environment(name, default):
+    """ This function looks up an envionment variable ``{{ name }}_ADDRESS``.
+    If it's defined, it is returned unmodified. If it's undefined, an environment
+    variable ``HOST_{{ name }}`` is looked up and resolved to an ip address.
+    If this is also not defined, the default is resolved to an ip address.
+    """
+    if "{}_ADDRESS".format(name) in environ:
+        return environ.get("{}_ADDRESS".format(name))
+    return resolve_address(environ.get("HOST_{}".format(name),default))
```
Comment (Nebukadneza) [inline socrate/system.py]: incredible nitpick: space after comma :sweat_smile: . I almost feel sorry for pointing out.

**Review by Nebukadneza — APPROVED** [review]
LGTM

File: socrate/system.py (hunk 1 above)
Comment (micw) [reply socrate/system.py]: No worries, fixed.

**Comment by kaiyou** [comment]
Looks good indeed :) i'll meege now and publish socrate tomorrow evening

**Pull request merged**